lib/src/components/NxTable/__tests__/NxTable.test.tsx

```diff
@@ -85,82 +91,369 @@ describe('NxTable', function() {
       </NxTable>
     );
 
-    expect(component.find('td')).toHaveProp('colSpan', 2);
-    expect(component.find('td')).toHaveClassName('nx-cell--meta-info');
+    expect(screen.getByRole('cell')).toHaveAttribute('colSpan', '2');
+    expect(screen.getByRole('cell')).toHaveClass('nx-cell--meta-info');
```

Comment:
This is a presentational class; no need to test

Reply:
> This is a presentational class; no need to test

Removed here:
56b1813e19885eff729fc247da16d0e96c8571d2
d696c91b16f059c37753530144ee44115de00a36